### PR TITLE
Bring disconnected landing page in line with new site content

### PR DESCRIPTION
**Pull request by richardmuscat**
#### Changes proposed in this Pull Request:

* Content changes to disconnected landing page

#### Testing instructions:

* Should look like this:

disconnected

### CODE REVIEW

File: _inc/client/components/jetpack-connect/index.jsx [hunk 1]
```diff
@@ -27,25 +27,78 @@ const JetpackConnect = React.createClass( {
...
+					</header>
+
+					<div className="jp-jetpack-connect__interior-container">
+						<img src={ imagePath + 'feature-photon-med.jpg' }
+							srcSet={ `${imagePath}feature-photon-sm.jpg 600w, ${imagePath}feature-photon-med.jpg 770w, ${imagePath}feature-photon-lrg.jpg 1200w` }
+							className="jp-jetpack-connect__feature-image" alt="Jetpacks photon serves up lightning fast, optimized images" />
+
+						<div className="jp-jetpack-connect__feature-list">
+							<div className="jp-jetpack-connect__feature-list-column">
+								<h3 title="Jetpack's WordPress themes" className="dops-section-header__label">
```
Comment (jeherve) [inline _inc/client/components/jetpack-connect/index.jsx]: Shouldn't we make those titles translatable as well?

File: _inc/client/components/jetpack-connect/index.jsx [hunk 2]
```diff
@@ -56,141 +109,97 @@ const JetpackConnect = React.createClass( {
...
 									<p>
-										{ __( 'Use Publicize to automatically share your posts with friends, followers, and the world.' ) }
+										{ __( 'Keep an eye on your success with simple, concise, and mobile-friendly stats.' ) }
 									</p>
 								</div>
 							</div>
 							<div className="jp-jetpack-connect__feature-list-column">
-								<h3 title="Jetpack's Sharing and Like features" className="dops-section-header__label">
-									{ __( 'Sharing & Like Buttons' ) }
+								<h3 title="Jetpack's publicize features" className="dops-section-header__label">
```
Comment (jeherve) [inline _inc/client/components/jetpack-connect/index.jsx]: Should we capitalize Publicize?

File: _inc/client/components/jetpack-connect/index.jsx [hunk 3]
```diff
@@ -56,141 +109,97 @@ const JetpackConnect = React.createClass( {
...
-						<h2 className="jp-jetpack-connect__container-subtitle" title="Site security and peace of mind with Jetpack">
-							{ __( 'Site security and peace of mind' ) }
+						<h2 className="jp-jetpack-connect__container-subtitle" title="Keep your site safe, 24/7">
+							{ __( 'Keep your site safe, 24/7' ) }
 						</h2>
 						<p className="jp-jetpack-connect__description">
 							{ __(
-								'Jetpack blocks malicious log in attempts, lets you know if your site goes down, ' +
-								'and can automatically update your plugins, so you don’t have to worry.'
+								'Automatic defense against hacks, malware, spam, data loss, and downtime.'
```
Comment (jeherve) [inline _inc/client/components/jetpack-connect/index.jsx]: Should it be "defends" here?

File: _inc/client/components/jetpack-connect/index.jsx [hunk 4]
```diff
@@ -56,141 +109,97 @@ const JetpackConnect = React.createClass( {
...
 							) }
 						</p>
 					</header>
 
 					<div className="jp-jetpack-connect__interior-container">
 						<div className="jp-jetpack-connect__feature-list">
 							<div className="jp-jetpack-connect__feature-list-column">
-								<h3 title="Jetpack's Protect feature" className="dops-section-header__label">
-									{ __( 'Protect', { context: 'Header. Noun: Protect is a module of Jetpack.' } ) }
+								<h3 title="Jetpack's monitor feature" className="dops-section-header__label">
```
Comment (jeherve) [inline _inc/client/components/jetpack-connect/index.jsx]: Should we capitalize Monitor here?

**Comment by jeffgolenski** [comment]
@richardmuscat looks good for a quick fix. BUT I think we should put on our list to update this page to reflect the design of the new jetpack.com homepage—or something a little more simple than the product tour page.  

01 welcome to jetpack - plugins splash

Here's a mock that @joanrho did for the full screen connection prompt on the /plugins/ page. Would be great to have something similar. cc @rickybanister

**Comment by rickybanister** [comment]
Just to add to what @jeffgolenski said, we would implement the same content for the disconnected landing page as we would for the plugins splash page. Same code, same content. Less to maintain. 

@johnHackworth's team may be able to work on that for the next release. 

I think it's fine to merge this PR for the next release, but it may be updated for the next one.

**Comment by zinigor** [comment]
@jeherve I have addressed your comments, can you please take a look again?

**Pull request merged**